niworkflows/interfaces/masks.py

```diff
@@ -189,11 +189,11 @@ def _post_run_hook(self, runtime):
         ''' generates a report showing slices from each axis '''
 
         self._anat_file = self.inputs.realigned_file
-        self._mask_file = self.aggregate_outputs().high_variance_mask
-        self._seg_files = [self.aggregate_outputs().high_variance_mask]
+        self._mask_file = self.aggregate_outputs().high_variance_masks
+        self._seg_files = [self.aggregate_outputs().high_variance_masks]
```

Comment:
Yeah, maybe just throw an informative exception so that if it ever comes up we won't have to spend any time figuring it out.